code/modules/atmospherics/gasmixtures/gas_mixture.dm

```diff
@@ -114,7 +114,7 @@ GLOBAL_LIST_INIT(gaslist_cache, init_gaslist_cache())
 		var/cached_gases = gases
 		TOTAL_MOLES(cached_gases, .)
 		. *= R_IDEAL_GAS_EQUATION * temperature / volume
-		return
+		return .
```

Comment:
For the record this is very very very very technically slower. It's alright here though.
That said, this syntax still makes me want to barf. do this instead
```
return . * R_IDEAL_GAS_EQUATION * temperature / volume 
```

Reply:
also something something atmos code is bad on purpose